python/lib/communication/dmod/communication/dataset_management_message.py

```diff
@@ -389,3 +361,87 @@ def is_awaiting(self) -> bool:
         """
         return self.data[self._DATA_KEY_IS_AWAITING]
 
+
+class MaaSDatasetManagementMessage(MaaSRequest, DatasetManagementMessage):
```

Comment:
Just a consistency note, that I think I have brought up before, but do we essentially use MaaS and DMOD "interchangeably", does it really matter, I don't know...

Reply:
Agreed; tracked by #171.